rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/NodeSource.java

```diff
@@ -94,6 +91,35 @@
 @ActiveObject
 public class NodeSource implements InitActive, RunActive {
 
+    public enum AdditionalInformation {
+
+        ACTUAL_COST,
+        COST_LIMIT
+    }
+
+    private Map<AdditionalInformation, String> additionalInformations = null;
```

Comment:
additional information should be generic and a Map<String, String>

Reply:
Ok